src/sec_edgar_bulker/core/downloader.py

```
@@ -0,0 +1,500 @@
+"""Main downloader for SEC EDGAR Bulker."""
+
+import asyncio
+import aiohttp
+import aiofiles
+import json
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Optional, Any, Union
+from tqdm import tqdm
+
+from .models import Config, Submission, Document, DocumentMetadata, ExcludedDocument
```

Ruff (0.8.2)

`.models.Document` imported but unused

Remove unused import

(F401)